### DISCUSSION TITLE
v0.3.0

### DISCUSSION THREAD

**Discussion by VoVAllen** [tensorchord/pgvecto.rs#506] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/tensorchord/pgvecto.rs/releases/tag/v0.3.0'>v0.3.0</a>.</em>

**Comment by onichandame** [comment]
anything worth mentioning when upgrading from 0.2.x to 0.3.0?

**Comment by Klench** [comment]
I updated from `pg16-v0.2.0` to `pg16-v0.3.0` (because I am stupid) but it made Immich stop working. I tried to revert back to v0.2.0 but the log now states the following:

> PostgreSQL Database directory appears to contain a database; Skipping initialization

What can I do?

**Reply by Klench** [reply]
Update: I managed to restore the database by following the [Immich docs](https://immich.app/docs/administration/backup-and-restore/#manual-backup-and-restore).
